src/nvim/testdir/test_functions.vim

```diff
@@ -875,6 +875,34 @@ func Test_byte2line_line2byte()
   bw!
 endfunc
 
+func Test_byteidx()
```

Comment:
Ideally, how should this test be included?

It's a new-style version of a test in test69 that was already lua'd here: https://github.com/neovim/neovim/blob/35041432bff6812126e1cc04d30d35a5c9117537/test/functional/legacy/069_multibyte_formatting_spec.lua#L309

Difference though is that this test uses method call syntax, which is why I'd like to include it.

As the patch removed the old version of the test, should I remove the lua'd version?
Should I keep both versions of the test? Should I translate this to a lua test? :shrug:

Reply:
I'll keep both in this case. the lua test tests the actual functionality, and this one for the `->` shenanigans